src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -101,7 +101,7 @@ <h3 id="history">
                   </div>
               {% endif %}
               {% if version.autoapprovalsummary %}
-                <div class="file-weight"><strong>{{ _('Weight:') }}</strong> {{ version.autoapprovalsummary.weight }}</div>
+                <div class="file-weight" title="{{ version.autoapprovalsummary.get_pretty_weight_info()|join('\n') }}"><strong>{{ _('Weight:') }}</strong> {{ version.autoapprovalsummary.weight }}</div>
```

Comment:
Not essential, but I'd be interested to see a screenshot of what this looks like.  (If it's too much hassle doesn't matter - it can wait until it's live on addons-dev)

Reply:
Ask and you shall receive! 
![risk_exposition](https://user-images.githubusercontent.com/187006/55420952-77736b00-5578-11e9-95b9-91050be925ce.png)

Note that yes, the keys of the dict are directly exposed without a translation, but I thought it was enough for now. We might want prettier strings later.